### ISSUE TITLE
Do we have Chained Scheduler for mmcv?

### ISSUE THREAD

**Issue by doem97** [open-mmlab/mmcv#2162]
Hi guys, thanks for contributions!
May I know if we have things like [torch.optim.lr_scheduler.ChainedScheduler](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.ChainedScheduler.html#torch.optim.lr_scheduler.ChainedScheduler) that combines multiple schedulers when tuning model? It is quite helpful and easy to extend.

Thanks!

**Comment by HAOCHENYE** [comment]
Hi~ mmcv does not have the function now.

**Comment by doem97** [comment]
> Hi~ We will support this feature in september, stay tuned!

Great, thanks!
